bin/burden

```diff
@@ -4104,14 +4029,14 @@ fi
 # Does not return
 #
 if [ $gl_tf_terminate -eq 1 ]; then
-	tf_terminate
+	source $UTILS_DIR/terraform_ops --terminate_list $gl_tf_term_list
```

Comment:
Does this need to be `source`'d in?

Reply:
Yes we want burden to terminate when the script is done.  We could duplicate the logic, but  then would have to deal with rtcs coming back.

Reply:
Wouldn't it make more sense to have this logic as a function, then source the script as if it's a library and call the function?

Reply:
Part of the idea, is these are also standalone.  So you are able to call terraform_ops outside of burden.

Reply:
If it was truly standalone, it would be essentially a script you call, check the exit code of, and handle appropriately.  As it is now, it relies on stuff within Zathras (`cleanup_and_exit`)

Reply:
Let's nudge this one.

I think it would be clearer for this to be it's own standalone script that can be used elsewhere.